process/work_tracking/milestones.md

```diff
@@ -0,0 +1,36 @@
+# Assigning issues and pull requests to milestones
+
+This document captures the process of when and how to assign a milestone to an issue or pull request (PR).
+
+## Types of milestones
+
+Our repos typically have a milestone that tracks an upcoming version to be released of the product tracked by the repo. Some repos may have more granular milestones leading up to the next minor/major version of the product.
+
+The milestones can be divided into three categories:
+
+* Patch milestone: A patch milestone tracks a patch release of the product which is meant to only contain bug fixes and no new features. The patch milestone is typically named following a pattern where it has a prefix referring to the name of the product followed by a version number `X.Y.Z`. The `Z` number refers to the patch version number which is incremented as part of the new release. It is also possible for a patch milestone to track a release candidate version other than the initial `rc1` release; in this case the version would be something like `X.Y.0-rc2`.
```

Comment:
nitpick, what do you think about removing the the last sentence on release milestones, and simply referring to the 3rd and final bullet point.

Reply:
I realized that a release milestone is not necessary. In my latest commit I simplified the document a little bit by removing it.

Reply:
LGTM